src/lua/call.c

```diff
@@ -617,7 +621,9 @@ static int read_cb(lua_State*L)
   FD_ZERO(&fdset);
   FD_SET(myfileno, &fdset);
   dt_lua_unlock();
+  #if !defined(_WIN32)
```

Comment:
Just removing the select will probably break the function.

@boucman: Do you have any insight what that function is used for and if removing the select is bad?